### DISCUSSION TITLE
FeynCalc amplitude squared giving polarisation vector with external momentum as its indices.

### DISCUSSION THREAD

**Discussion by scsa007316** [FeynCalc/feyncalc#236] [General]
I calculated the amplitude for a process using feynarts and feyncalc. And in the squared amplitude expression it is giving terms which have polarisation vectors with external momentum as its indices. Now I know this result does not make sense something went wrong somewhere.
Here is what I did, I calculated the amplitude using " amp[0] = 
 FCFAConvert[CreateFeynAmp[if1], IncomingMomenta -> {p1, p2}, 
  OutgoingMomenta -> {k1, k2, k3, k4}, UndoChiralSplittings -> True, 
  ChangeDimension -> 4, List -> False, SMP -> True, Contract -> True]"
and stored it in amp[0], this result was good it made sense. Then I computed squared amplitude using - " ampSquared[0] = 
 Simplify[
  DiracSimplify[(FermionSpinSum[#1, ExtraFactor -> 1/4] &)[
    FeynAmpDenominatorExplicit[amp[0]*camp[0]]]]]" and it gave terms like Overscript[\[Epsilon], _]^(Overscript[k1, _]Overscript[k2, _]Overscript[k4, _]Overscript[p1, _]) (Overscript[k1, _]\[CenterDot]Overscript[p2, _]) (Overscript[k1, _]\[CenterDot]Overscript[k3, _]).

The process considered is 2 to 4 process. Any help/suggestions?

**Comment by vsht** [comment]
My suggestion would be to read the docu first, especially the tutorial part that explains the notation

<https://feyncalc.github.io/FeynCalcBookDev/InternalExternal.html>



Am 25. Oktober 2023 08:14:59 UTC schrieb Aadarsh singh ***@***.***>:

**Reply by scsa007316** [reply]
That was straight to the point suggestion. Thanks.

**Comment by scsa007316** [comment]
Query Resolved. It was my bad.
